uxarray/grid.py

```diff
@@ -284,21 +291,24 @@ def integrate(self, var_key):
 
     @property
     def face_areas(self):
-        """ Face area calculation property for grid class, calculates area of all faces in the mesh
+        """Face area calculation property for grid class, calculates area of
+        all faces in the mesh.
+
         Returns
         -------
 
-        ndarray: area of all the faces in the mesh.
+        area of all the faces in the mesh. : ndarray
 
-        Examples
-        --------
```

Comment:
"Examples" is the standard: https://numpydoc.readthedocs.io/en/latest/format.html#examples